ros_main:
  ros__parameters:
    transport: "udp"
    scanner_ip: "10.0.10.76"
    port: "80" # optional
    device: "R2000"
    start_angle: -1800000
    max_num_points_scan: 0
    samples_per_scan: 2800
    scan_frequency: 15
    packet_type: "C"
    scan_topic: "/scan"
    frame_id: "front_laser"
    watchdogtimeout: 60000
    watchdog: true
